addons/binding/org.openhab.binding.amazonechocontrol/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: AmazonEchoControl Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.amazonechocontrol;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Export-Package: 
+ org.openhab.binding.amazonechocontrol,
+ org.openhab.binding.amazonechocontrol.handler
+Import-Package: 
+ org.apache.commons.lang.time;version="2.6.0",
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.openhab.binding.amazonechocontrol,
+ org.openhab.binding.amazonechocontrol.handler,
+ org.openhab.core.library.types,

```

Comment:
This dependency causes problems, please refactor to no be dependent on it anymore.

Reply:
Hi Martin, I do not what you mean with remove these dependencies, because the are used, but I have copied now the dependencies from the icloud binding (I only changed the names from icloud with amazonechocontrol). Now Jenkis is green again. But the build goes red again, but it has produced my binding: https://openhab.jfrog.io/openhab/libs-pullrequest-local/org/openhab/binding/org.openhab.binding.amazonechocontrol/2.3.0-SNAPSHOT/ 
I do not know what's wrong. Thanks for your help!